cfme/networks/cloud_network.py

```diff
@@ -42,7 +43,10 @@ def exists(self):
     def parent_provider(self):
         """ Return object of parent cloud provider """
         view = navigate_to(self, 'Details')
-        provider_name = view.entities.relationships.get_text_of('Parent ems cloud')
+        parent_cloud_text = VersionPick({Version.lowest(): 'Parent ems cloud',
```

Comment:
OPTIONAL: I don't think you need to resolve VersionPick here since widgetastic is able to resolve VersionPicks automatically.